doc/cookbook/source/examples/converter/diffusion_maps.rst

```diff
@@ -0,0 +1,37 @@
+==============================
+Diffusion Maps
+==============================
+
+Diffusion Maps is a non-linear dimensionality reduction method that uses 
+eigenfunctions of Markov matrices to construct coordinates called diffusion maps 
+that can be used to generate efficient representations of complex geometric structures.
```

Comment:
Is there a key equation that we could show here? You could shop around the web to see what other libraries use to describe this